templates/pages/movie.html.twig

```diff
@@ -0,0 +1,161 @@
+                                    </g>
+                                </svg>
+                            {% endfor %}
+                            <h3 class="ml-2 text-sm font-semibold dark:text-white">{{ review.title }}</h3>
+                        </div>
+                        <div class="mb-5 text-sm text-gray-500 dark:text-gray-400">
+                            <p>Reviewed on {{ review.createdAt|date('j F, Y') }}</p>
+                        </div>
+                        <p class="mb-2 dark:text-gray-400">{{ review.content }}</p>
+                    </div>
+                {% endfor %}
+                {% include "components/pagination.html.twig" %}            
+            {% endif %}
+        </div>
+
+        {# Rate menu #}
+        <div class="relative z-10 text-white" aria-labelledby="modal-title" role="dialog" aria-modal="true" 
+            x-show="rateMenuOpen"
+        >
+            <div class="fixed inset-0 bg-gray-500 bg-opacity-75 transition-opacity"></div>
+
+            <div class="fixed inset-0 z-10 w-screen overflow-y-auto">
+                <div class="flex min-h-full items-end justify-center p-4 text-center sm:items-center sm:p-0">
+                    <div class="relative transform overflow-hidden rounded-lg bg-gray-900 px-4 pb-4 pt-5 text-left shadow-xl transition-all sm:my-8 sm:w-full sm:max-w-sm sm:p-6 flex flex-col items-center"
+                        @click.outside="rateMenuOpen = false"
+                    >
+                        <p class="text-gray-300">{{ movie.title }}</p>
+                        {% include "components/rate-menu.html.twig" %}
+                        <div class="mt-5 sm:mt-6">
+                            <button type="button" class="inline-flex w-full justify-center rounded-md bg-indigo-600 px-6 py-3 text-lg font-semibold text-white shadow-sm hover:bg-indigo-500 focus-visible:outline focus-visible:outline-2 focus-visible:outline-offset-2 focus-visible:outline-indigo-600">
+                                Rate
+                            </button>
+                        </div>
+                    </div>
+                </div>  
+            </div>
+        </div>
+
+        {# Review menu #}
```

Comment:
I am confused with working 'menu'. Is it a dialog? Then it should be a dialog or modal